src/main/java/zmaster587/advancedRocketry/client/render/planet/RenderPlanetarySky.java

```diff
@@ -343,9 +340,10 @@ public void render(MatrixStack matrix, float partialTicks) {
 		float f6;
 
 		//Simulate atmospheric thickness
-		f1 *= atmosphere;
-		f2 *= atmosphere;
-		f3 *= atmosphere;
+		f1 = properties.getAtmosphereDensity() < 1 ? 0 : (float) Math.pow(f1, Math.sqrt(Math.max(atmosphere, 0.81)));
```

Comment:
Could we set `properties.getAtmosphereDensity()` to a variable like we do `atmosphere` once to avoid calling it 3x?

If we're using absolute atmoshperic density here it might be good to rename `atmosphere` to `atmosphereAtHeight` or something and create a new `atmosphere` variable